protocol/avdl/stellar1/remote.avdl

```diff
@@ -166,4 +166,16 @@ protocol remote {
   boolean isMasterKeyActive(keybase1.UserVersion caller, AccountID accountID);
 
   string ping();
+
+  // stellar.payment_status gregor message body
+  record PaymentStatusMsg {
+    KeybaseTransactionID keybaseID;
+    TransactionID stellarID;
+    boolean pending;
+    boolean relay; // Is it a relay payment?
+    // After relay payment was submitted, was there already a
+    // claimant user?
+    boolean autoClaimed;
+    union { null, string } errMsg;
+  }
```

Comment:
Move to stellar1/gregor.avdl to just to stay organized.

Reply:
This is a sprawling combination of fields for a notification that we don't know yet how it will be used. Seems simpler to either just send the IDs so that the client can call up to refresh, or send down a full Summary type. For now I'd recommend just sending kbTxID and txID (rename them). We can add more later if needed. So the message just means 'something has changed'.